www/caddy/pkg-descr

```diff
@@ -26,6 +26,10 @@ DOC: https://docs.opnsense.org/manual/how-tos/caddy.html
 Plugin Changelog
 ================
 
+1.5.5_1
+
+* Fix: The newly introduced "configctl caddy reload" action, which calls the "service caddy reloadssl" command, will now also trigger the setup.sh script.
```

Comment:
just add this to 1.5.5 -- I've stopped doing release notes for revision bumps if they fix something only

Reply:
Sure, that's easy to fix. ^^